components/card/index.css

```diff
@@ -10,7 +10,79 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+  --spectrum-card-title-font-size: var(--spectrum-heading-size-xxs);
+  --spectrum-card-title-font-weight: var(--spectrum-heading-sans-serif-font-weight);
+  --spectrum-card-title-font-style: var(--spectrum-heading-sans-serif-font-style);
+  --spectrum-card-title-line-height: var(--spectrum-heading-line-height);
+  --spectrum-card-title-color: var(--spectrum-heading-color);
+
+  --spectrum-card-body-font-family: var(--spectrum-sans-font-family-stack);
+  --spectrum-card-body-font-size: var(--spectrum-body-size-s);
+  --spectrum-card-body-font-weight: var(--spectrum-body-sans-serif-font-weight);
+  --spectrum-card-body-font-style: var(--spectrum-body-sans-serif-font-style);
+  --spectrum-card-body-line-height: var(--spectrum-body-line-height);
+  --spectrum-card-body-font-color: var(--spectrum-body-color);
+
+  /* Quick Actions */
+  --spectrum-card-actions-spacing: var(--spectrum-spacing-300);
+  --spectrum-card-actions-size: var(--spectrum-card-selection-background-size);
+  --spectrum-card-actions-border-radius: var(--spectrum-corner-radius-100);
+  /* TODO update to --spectrum-card-selection-background-color token once an RGB stripped value is available */
+  --spectrum-card-actions-background-color-rgb: var(--spectrum-gray-100-rgb);
+  --spectrum-card-actions-background-color-opacity: var(--spectrum-card-selection-background-color-opacity);
+  --spectrum-card-actions-drop-shadow-color: var(--spectrum-drop-shadow-color);
+  --spectrum-card-actions-drop-shadow-x: var(--spectrum-drop-shadow-x);
+  --spectrum-card-actions-drop-shadow-y: var(--spectrum-drop-shadow-y);
+  --spectrum-card-actions-drop-shadow-blur: var(--spectrum-drop-shadow-blur);
+
+  /* Focus */
+  --spectrum-card-focus-indicator-gap: var(--spectrum-focus-indicator-gap);
+  --spectrum-card-focus-indicator-color: var(--spectrum-focus-indicator-color);
+  --spectrum-card-focus-indicator-width: var(--spectrum-focus-indicator-thickness);
+  --spectrum-card-focus-indicator-corner-radius: var(--spectrum-corner-radius-100);
+
+  /* Quiet */
+  --spectrum-card-quiet-background-color: var(--spectrum-background-base-color);
+  --spectrum-card-quiet-background-color-hover: var(--spectrum-gray-300);
+  --spectrum-card-quiet-header-margin-top: var(--spectrum-spacing-100);
+  --spectrum-card-quiet-minimum-width: var(--spectrum-card-minimum-width);
+  --spectrum-card-folder-background-color: var(--spectrum-gray-300);
+  --spectrum-card-file-background-color: var(--spectrum-gray-50);
+  --spectrum-card-file-outline: var(--spectrum-gray-500);
+
+  /* Horizontal */
+  --spectrum-card-horizontal-body-padding: var(--spectrum-spacing-300);
+  --spectrum-card-horizontal-preview-padding: var(--spectrum-spacing-200);
+
+}
+
+.spectrum-Card--sizeS {
+  /* Need less spacing for small cards since we've decreased min-width */
+  --spectrum-card-actions-spacing: calc(var(--spectrum-spacing-300) / 2);
+
+}
 
 .spectrum-Card--sizeS {
```

Comment:
We'll need to remove the @remapvars calls here and in `.spectrum-Card--sizeM`, too. This was functionality for the older "vars" tokens system that we're no longer using.

GitHub wasn't allowing me to highlight and call-out the code here (I guess because it wasn't changed?) but I'm thinking we should remove lines 87 - 101.